fvm/transactionContractFunctionInvocator.go

```diff
@@ -53,6 +53,25 @@ func (i *TransactionContractFunctionInvoker) Invoke(env Environment, parentTrace
 		defer span.Finish()
 	}
 
+	switch e := env.(type) {
+	case *TransactionEnv:
+		if e.sth.EnforceComputationLimits {
+			err := e.sth.State().MeterComputation(meter.ComputationKindContractFunctionInvoke, 1)
+			if err != nil {
+				return nil, err
+			}
+		}
+	case *ScriptEnv:
+		if e.sth.EnforceComputationLimits {
+			err := e.sth.State().MeterComputation(meter.ComputationKindContractFunctionInvoke, 1)
+			if err != nil {
+				return nil, err
+			}
+		}
```

Comment:
this should be only transactionContractFunctionInvocator so it shouldn't be any need to disambiguate here, if there is some call from scriptsEnv to this, there should be a copy of this with script prefix.